builtin/providers/azurerm/resource_arm_network_interface_card.go

```diff
@@ -252,6 +258,17 @@ func resourceArmNetworkInterfaceRead(d *schema.ResourceData, meta interface{}) e
 		}
 	}
 
+	if iface.IPConfigurations != nil && len(*iface.IPConfigurations) > 0 {
+		var privateIPAddress *string
+		for _, ipConfig := range *iface.IPConfigurations {
+			privateIPAddress = ipConfig.Properties.PrivateIPAddress
```

Comment:
I hit the same problem as last time.

```
builtin/providers/azurerm/resource_arm_network_interface_card.go:264: 
invalid operation: iface.IPConfigurations[0] (type *[]network.InterfaceIPConfiguration does not support indexing)
```

Should I assign the pointer value to a variable and try like that?